frontend/public/components/utils/kebab.tsx

```diff
@@ -108,40 +108,80 @@ export const ResourceKebab = connectToModel((props: ResourceKebabProps) => {
   />;
 });
 
-export class Kebab extends DropdownMixin {
+export class Kebab extends React.Component<KebabProps, KebabState> {
   static factory: KebabFactory = kebabFactory;
-  private onClick = this.onClick_.bind(this);
+  constructor(props) {
+    super(props);
+    this.state = {
+      isOpen: false,
+    };
+  }
 
-  onClick_(event, option) {
-    event.preventDefault();
+  onToggle = isOpen => {
+    this.setState({
+      isOpen,
+    });
+  };
 
-    if (option.callback) {
-      option.callback();
-    }
+  onSelect = () => {
+    this.setState({
+      isOpen: !this.state.isOpen,
+    });
+  };
 
-    if (option.href) {
-      history.push(option.href);
+  render() {
+    const { isOpen } = this.state;
+    const { options, isDisabled, position, id, children } = this.props;
+
+    let items = [];
+    if (options && options.length){
+      options.forEach((option) => {
+        items.push(
+          <DropdownItem key={option.label}
+            isDisabled={isDisabled}
+            onClick={option.callback}
+            href={option.href}
```

Comment:
This causes a full page refresh when clicking the links because we're not using `<Link>`

Reply:
just fixed it ;)